jbmc/src/java_bytecode/expr2java.cpp

```diff
@@ -422,11 +422,10 @@ std::string expr2javat::convert_with_precedence(
     return "pod_constructor";
   else if(src.id()==ID_virtual_function)
```

Comment:
`can_cast_expr`?

Reply:
This is in a long list of checks on `src.id()`, so I thought it better not to have one inconsistent one